elfi/bo/acquisition.py

```diff
@@ -42,15 +42,15 @@ def evaluate(self, x, t=None):
         """
         return NotImplementedError
 
-    def acquire(self, n_values, pending_locations=None, t=None):
+    def acquire(self, n_values, pending_locations=None, t=0):
```

Comment:
It could be None as well, but then there should be a check. Otherwise it crashes if t is not given.